src/main/java/ip/src/main/java/Level_1.java

```diff
@@ -0,0 +1,19 @@
+package ip.src.main.java;
+import java.util.Scanner;
+
+public class Level_1 {
```

Comment:
Maybe consider deleting dead code? So that your codebase can be simpler and easier to understand.